persistqueue/queue.py

```diff
@@ -138,6 +138,9 @@ def qsize(self):
     def _qsize(self):
         return self.info['size']
 
+    def empty(self):
```

Comment:
Can you please name it to `is_empty`, empty maybe treat as `clear my clear` by some people.

Reply:
Hello @peter-wangxu, I called them `#.empty` so they are consistent with python's queues: https://docs.python.org/3/library/queue.html#queue.Queue.empty so that some code can transparently use python's queues or yours. But I can change it if you feel sticking to python's - admittedly weird - naming is unnecessary. What do you think?

Reply:
OK, let's follow the convention.